libs/eslint-plugin/README.md

```diff
@@ -1,6 +1,6 @@
 # @rx-angular/eslint-plugin
 
-> A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
+A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
```

Comment:
question: Any reason for this change?

suggestion: Revert the change to leave this file untouched in this PR.
```suggestion
> A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
```

Reply:
Oops - no I guess this happened by accident. Good catch!